packages/mds-utils/utils.ts

```diff
@@ -770,6 +772,44 @@ function normalizeToArray<T>(elementToNormalize: T | T[] | undefined): T[] {
   return [elementToNormalize]
 }
 
+export interface ProviderClaimResponse extends ApiResponse {
+  locals: ApiResponseLocals & {
+    provider_id: UUID
+  }
+}
+
+async function providerClaimMiddleware(req: ApiRequest, res: ProviderClaimResponse) {
```

Comment:
This _isn't_ standard Express middleware format. The signature should include a third `next: NextFunction()` argument. When successful, this middleware should `return next()` ... when there is an error it should `return res.status(...)`